spyderlib/config.py

```diff
@@ -510,6 +510,17 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Alt+B",
+              'editor/next char': "Alt+F",
+              'editor/previous word': "Meta+B",
+              'editor/next word': "Meta+F",
+              'editor/Kill to Line End': "Meta+K",

```

Comment:
Ahem: "C-k: kill from cursor to the end of the line", from the exact docs you pointed to :stuck_out_tongue: